src/main/java/org/unibl/etf/epraksa/repositories/ReportByMentorRepository.java

```diff
@@ -0,0 +1,13 @@
+package org.unibl.etf.epraksa.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.unibl.etf.epraksa.model.entities.ReportByMentor;
+
```

Comment:
selektuj samo da ti vrati citav report kao objekat
select internship.reportByMentor from StudentHasInternship shi inner join fetch shi.reportByMentor where shi.id.stidentId= :id and shi.id.internshipId= : internshipId   //.id ide zbog embedded PK klase